README.md

```diff
@@ -134,26 +134,9 @@ Now, you have everything ready to start the installation of FOSSBilling,. Naviga
 
 This guide assumes you already have [Docker](https://docs.docker.com/get-docker/), [Git](https://git-scm.com) and [GNU make](https://www.gnu.org/software/make/) installed.
```

Comment:
This line talked about a guide that will no longer exist after this PR

Reply:
If we remove it from here, I think we should add it to the docker repo.

Reply:
That should be fine it doesn't belong here